packages/chain-adapters/src/evm/EvmBaseAdapter.ts

```diff
@@ -218,7 +218,7 @@ export abstract class EvmBaseAdapter<T extends EvmChainIds> implements IChainAda
     const addressNList = bip32ToAddressNList(path)
     const address = await (wallet as ETHWallet).ethGetAddress({
       addressNList,
-      showDisplay: showOnDevice
+      showDisplay: Boolean(showOnDevice)
```

Comment:
Thanks for the fix on this. Can I suggest we use a default value for `showOnDevice` when destructuring instead of the `Boolean()` cast to make this more clear and explicit?

Reply:
I am okay with that.